metricbeat/etc/kibana/visualization/Avg.-Total-Memory.json

```diff
@@ -0,0 +1,11 @@
+{
+  "visState": "{\"title\":\"Avg. Total Memory\",\"type\":\"metric\",\"params\":{\"handleNoResults\":true,\"fontSize\":\"40\"},\"aggs\":[{\"id\":\"1\",\"enabled\":true,\"type\":\"avg\",\"schema\":\"metric\",\"params\":{\"field\":\"system.memory.total\",\"customLabel\":\"Total Memory\"}}],\"listeners\":{}}", 
```

Comment:
The file name with `Avg.` is interesting, but that is more something related to Kibana and brings up the question if we should use abbreviations in the title.